specification/powerbidedicated/Powerbidedicated.Management/models.tsp

```diff
@@ -0,0 +1,692 @@
+  Pausing: "Pausing",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Resuming: "Resuming",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Preparing: "Preparing",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Scaling: "Scaling",
+}
+
+/**
+ * Specifies the generation of the Power BI Embedded capacity. If no value is specified, the default value 'Gen2' is used. [Learn More](https://docs.microsoft.com/power-bi/developer/embedded/power-bi-embedded-generation-2)
+ */
+union Mode {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Gen1: "Gen1",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Gen2: "Gen2",
+}
+
+/**
+ * The type of identity that created/modified the resource.
+ */
+union IdentityType {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  User: "User",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Application: "Application",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  ManagedIdentity: "ManagedIdentity",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Key: "Key",
+}
+
+/**
+ * The name of the Azure pricing tier to which the SKU applies.
+ */
+union VCoreSkuTier {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  AutoScale: "AutoScale",
+}
+
+/**
+ * The current deployment state of an auto scale v-core resource. The provisioningState is to indicate states for resource provisioning.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+union VCoreProvisioningState {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Succeeded: "Succeeded",
+}
+
+/**
+ * Represents the SKU name and Azure pricing tier for PowerBI Dedicated capacity resource.
+ */
+model CapacitySku {
```

Comment:
>CapacitySku [](http://example.com/codeflow?start=6&length=11)

nitpicky question....
why is it called a CapacitySku instead of just a Sku?

Reply:
Also now I think about it... 'SKU of the capacity' and 'capacity of the SKU' are unexpectedly contradictory meanings of the word capacity aren't they?

Why talk about 'SKU of the capacity' at all though? Why not just say 'SKU of the reservation' or 'amount of SKU reserved'?

Reply:
I presume we can't fix this unless its SDK breaking, which is not ideal. #wontfix sound good?

Reply:
Yeah we won't do any change for service team.